front/app/components/VotesBudget/index.tsx

```diff
@@ -0,0 +1,67 @@
+import React, { useMemo } from 'react';
+
+// api
+import useProjectById from 'api/projects/useProjectById';
+import usePhases from 'api/phases/usePhases';
+import useAppConfiguration from 'api/app_configuration/useAppConfiguration';
+import useBasket from 'api/baskets/useBasket';
+
+// i18n
+import useLocalize from 'hooks/useLocalize';
+import { useIntl } from 'utils/cl-intl';
+import messages from './messages';
+
+// utils
+import { getCurrentPhase, getLastPhase } from 'api/phases/utils';
+
+interface Props {
+  projectId: string;
+}
+
+const VotesBudget = ({ projectId }: Props) => {
```

Comment:
This isn't only used for budgeting though - in other types of voting the getVoteTerm() in this component returns the admin-selected vote term (in the cumulative voting front office branch I have an additional check to set currency vs the custom vote term)

Reply:
I think we could leave it like this, but rename it to "VotesCounter" or something maybe?